uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-actions.component.js

```diff
@@ -22,13 +23,16 @@ define([
       return sharedSettings.sharedPathTpl + 'components/leave-request-actions.html';
     }],
     controllerAs: 'actions',
-    controller: ['$log', '$q', '$rootScope', 'OptionGroup', 'dialog', 'shared-settings', controller]
+    controller: LeaveRequestActionsController
   });
 
-  function controller ($log, $q, $rootScope, OptionGroup, dialog, sharedSettings) {
+  LeaveRequestActionsController.$inject = ['$log', '$rootScope', 'dialog', 'pubSub', 'shared-settings'];
+
+  function LeaveRequestActionsController ($log, $rootScope, dialog, pubSub, sharedSettings) {
     $log.debug('Component: leave-request-action-dropdown');
 
     var vm = this;
+    var statusIdBeforeAction;
```

Comment:
But why? We only add line breaks after all vars are defined and before method definition has started.